javascript/javascript2/week2/hyfBayHelper/main.js

```diff
@@ -0,0 +1,61 @@
+
+  console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+
+
+const productsList = document.getElementById('productsList');
+ const search= document.getElementById('search-item');
+
+const searchPrice= document.getElementById('searchPrice')
+
+
+function renderProducts(products) {
+
+const productsList = document.querySelector('.productsList');
```

Comment:
you can move this productList variable into global scope, there is no need to have it in function ☺️ and add `productsList.innerHTML = "";`